styles.css

```diff
@@ -451,3 +443,286 @@ form {
   color: white;
   background-color: #7f8cff;
 }
+
+@media all and (max-width: 768px) {
+  #header .nav_icon ul {
+    display: none;
+  }
+
+  .for-big-size {
+    display: none;
+  }
+}
+
+/* desktop version */
+
+@media all and (min-width: 768px) {
+  #header {
+    padding: 0;
+    margin: 25px 0 0 155px;
+  }
+
+  /* page one */
+
+  .frame-2 ul {
+    margin-left: 0;
+  }
+
+  .tags ul {
+    margin-left: 0;
+  }
+
+  .language-list ul {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .language-list ul li {
+    flex-direction: column;
+    max-width: 122px;
+    max-height: 120px;
+    padding: 2% 7% 25% 3%;
+  }
+
+  #third-page ul {
+    margin: 0 auto 32px 48px;
+  }
+
+  #header .nav_icon ul {
+    display: flex;
+    list-style: none;
+    margin-right: 154px;
+  }
+
+  #header .nav_icon ul li {
+    margin-right: 32px;
+  }
+
+  #header .nav_icon ul li:last-of-type {
+    margin-right: 0;
+  }
+
+  #header .nav_icon a {
+    display: none;
+  }
+
+  #first_page {
+    background-image: url(images/Header\ bg@2x.png);
+    background-size: cover;
+  }
+
+  .cover {
+    background-image: none;
+  }
+
+  .title h1 {
+    margin-left: 0;
+    padding: 0;
+  }
+
+  #first_page h1 {
+    margin-left: 260px;
+    padding-top: 281px;
+    margin-bottom: 12px;
+  }
+
+  .upper-part p {
+    margin-left: 0;
+    margin-top: 12px;
+    max-width: 447px;
+  }
```

Comment:
- [ ] Kindly ensure that the color of your headline description text matches your chosen Figma design 👍🏽 . Kindly follow the design specification on Figma 👍🏽 

| Figma Design  | Your Design |
| ------------- | ------------- |
|![image](https://user-images.githubusercontent.com/63473584/194621748-e9472f1d-22b2-456c-9da8-8e6b9c787aae.png)  | ![image](https://user-images.githubusercontent.com/63473584/194621898-cef32cab-d010-4fc4-8ae1-2d9f1a979a84.png)  |